### ISSUE TITLE
Syncing rpc return None

### ISSUE THREAD

**Issue by hujw77** [darwinia-network/darwinia-common#850]
## Script
```
const Web3 = require('web3');
const web3 = new Web3('wss://pangolin-rpc.darwinia.network');

web3.eth.subscribe('syncing')
    .on("connected", function (subscriptionId) {
        console.log(subscriptionId);
    })
    .on("data", function (log) {
        console.log(log);
    });

```

## Error
This is no data respond
```
$ event node syncing.js
0x4469505568684d745779554445467968
```

**Comment by AurevoirXavier** [comment]
Same with #848?

**Comment by AsceticBear** [comment]
Not sure. I need to dig deeper into this issue.

**Comment by AsceticBear** [comment]
https://github.com/paritytech/frontier/pull/630 Waiting for the next frontier upgrade

**Comment by AsceticBear** [comment]
This had been fixed in the `darwinia-0.12.4` version. Close it.
